### PR TITLE
Update maintainer information

### PR DESCRIPTION
**Pull request by vinceatbluelabs**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,7 +3,11 @@ sqlalchemy-vertica-python
 
 Vertica dialect for sqlalchemy. Forked from the `Vertica ODBC dialect <https://pypi.python.org/pypi/vertica-sqlalchemy>`_.
 
-This module implements a Vertica dialect for SQLAlchemy using `vertica-python <https://github.com/uber/vertica-python>`_. Engine creation: 
+This module implements a Vertica dialect for SQLAlchemy using `vertica-python <https://github.com/uber/vertica-python>`_
```
Comment (cwegrzyn) [inline README.rst]: Should this point at the official vertica repo now?

File: README.rst [hunk 2]
```diff
@@ -19,7 +23,7 @@ From PyPI: ::
 
 From git: ::
 
-     git clone https://github.com/LocusEnergy/vertica-sqlalchemy 
+     git clone https://github.com/bluelabsio/vertica-sqlalchemy
```
Comment (cwegrzyn) [inline README.rst]:
```suggestion
     git clone https://github.com/bluelabsio/vertica-sqlalchemy-python
```

File: README.rst [hunk 3]
```diff
@@ -19,7 +23,7 @@ From PyPI: ::
...
      cd vertica-sqlalchemy
```
Comment (cwegrzyn) [inline README.rst]:
```suggestion
     cd vertica-sqlalchemy-python
```

**Review by cwegrzyn — APPROVED** [review]
Few small things, but looks good (assuming original author is okay with attribution)

File: README.rst (hunk 1 above)
Comment (vinceatbluelabs) [inline README.rst]:
```suggestion
This module implements a Vertica dialect for SQLAlchemy using `vertica-python <https://github.com/vertica/vertica-python>`_
```

Comment (vinceatbluelabs) [reply README.rst]: Done

**Comment by lemeryfertitta** [comment]
@vinceatbluelabs fine with the attribution, but, as far as I know, the original author was James Casbon (https://github.com/jamescasbon/vertica-sqlalchemy). I maintained the fork under the new PyPI name for a few years.

Thanks for taking this over guys. I will put a link in the README on the LocusEnergy fork to this one.

**Comment by lemeryfertitta** [comment]
I've archived the LocusEnergy repo and included a note in the README pointing here. Let me know if there is anything else I can help with for the transition.

**Comment by vinceatbluelabs** [comment]
Thanks, @lemeryfertitta!  I think we're in great shape here and I really appreciate your help with the transition. 

I updated the PR to credit both you and @jamescasbon.  This codebase enabled a big chunk of my work at BlueLabs in various ways over the last few years - thank you both!

**Pull request merged**